src/meta-srv/src/error.rs

```diff
@@ -710,6 +710,14 @@ pub enum Error {
         source: common_meta::error::Error,
     },
 
+    #[snafu(display("Peer count mismatch, expected: {}, actual: {}", expected, actual))]
+    PeerCountMismatch {
```

Comment:
duplicate with `NoEnoughAvailableNode`

Reply:
Resolved in [6f6b0c8](https://github.com/GreptimeTeam/greptimedb/pull/5990/commits/6f6b0c8cf0af29944c97c77c19d20822131aa0eb)